### ISSUE TITLE
logic behind extrusion widths?

### ISSUE THREAD

**Issue by firsttest12** [AndrewEllis93/Ellis-SuperSlicer-Profiles#17]
Eager to understand the logic behind the use of 

 - 1x (100%) extrusion width for "normal" parts: 0.4mm for a 0.4mm nozzle etc. ?

- 115% extrusion width for the perimeters of Decorative parts?

Simply because:

- it's commonly argued that, in order to get better binding between perimeters, it's better to have extrusion widths around 120%: because at 100% the extrudate is significiantly rounded (beause it has come out of a round aperture and has not been squished under that flat portion of the nozzle) at the edges (also layer height dependent). Would have expected that to be used for the "normal" (functional) profiles.

Additionally I've sort of noticed that extruding at 100% (1x) can give parts with a "cleaner" finish, that in some ways are therefore more aesthetic. ( For example prints seem to show less Z banding, perhaps because minor Z fluctuations are "absorbed" by the slightly curved edges of the outermost perimeter.) Might have expected 100% to be use for aesthetic parts.

In other words, genuinely confused.

**Comment by AndrewEllis93** [comment]
The PIF profile only has 100% external perims because it's a requirement for Voron parts - I don't actually like 100% widths, and took the opportunity to fix it in my decorative profiles. 

The internal PIF widths are wider to make up for strength. The 100% externals' poor stepover does generally work fine for Voron parts, as they avoid steep overhangs in the designs.

As for the decorative profiles' 115% widths:
- Partially in case someone is using variable layer heights and exceeds 50% layer heights
- For better stepover for overhangs. 
- Some people have really pointy nozzles though, so going past 115% has caused some complaints. And taking it too far can affect surface finish, as you say. 
- A middleground. Such are the woes of trying to create a "universal" profile - pretty much impossible. I change a lot of these settings print-to-print, but that's just something people have to learn through experience, really.

**Comment by firsttest12** [comment]
Thanks.

Voron's 1x logic seems to be lost in the mists of time. Odd.

In the meantime, thanks for the excellent - and thought-provoking - profiles!

**Comment by AndrewEllis93** [comment]
You're welcome!
